### PR TITLE
DS-4804 Fixup and add unit test

### PR DESCRIPTION
**Pull request by jrwishart**

### CODE REVIEW

File: tests/testthat/test-plotly.R [hunk 1]
```diff
@@ -358,3 +358,17 @@ test_that("Line breaks are properly translated (R -> HTML)", {
...
    expect_warning(group_by(plot_ly(txhousing), city), NA)
 })
+
+test_that("Check QTables dont cause errors", {
+    .createQTableArray <- function(n, class.name)
+        structure(
+            array(runif(n), dim = c(n, 1L)),
+            class = class.name
+        )
+    s <- data.frame(
```
Comment (mwmclean) [inline tests/testthat/test-plotly.R:363]: This results in `Error in as.data.frame.default(x[[i]], optional = TRUE) : 
  cannot coerce class ‘"QTable"’ to a data.frame`
for me with R v4.1.2 and v4.2.3

**Review by mwmclean — CHANGES_REQUESTED** [review]

File: tests/testthat/test-plotly.R (hunk 1 above)
Comment (jrwishart) [reply tests/testthat/test-plotly.R:363]: Thanks for checking, test at https://app.displayr.com/Dashboard?project_id=-1047844#page=69b42f15-377d-4a9f-930d-33f6308a06f8

Comment (mwmclean) [reply tests/testthat/test-plotly.R:363]: Still seeing the same error trying to create the `s` data.frame in the test.

Comment (jrwishart) [reply tests/testthat/test-plotly.R:363]: Should be fixed in b95fd2ffffc70bd31b90520d5074ddc74e11df2d

**Review by mwmclean — APPROVED** [review]

**Pull request merged**